lib/content_block_tools/presenters/base_presenter.rb

```diff
@@ -26,7 +26,7 @@ def initialize(content_block)
       # @return [string] A HTML representation of the content block
       def render
         content_tag(
-          self.class::BASE_TAG_TYPE,
+          field_names ? :span : self.class::BASE_TAG_TYPE,
```

Comment:
minor: could see this stray `:span` logic getting a bit lost - could it be a method in itself, and maybe the `:span` is a constant on the field presenter?